### DISCUSSION TITLE
How to clear all cartesian chart contents?

### DISCUSSION THREAD

**Discussion by En-JON-eer** [Live-Charts/LiveCharts2#710] [Q&A]
![image](https://user-images.githubusercontent.com/89985196/197544806-29c15b84-59f1-4c9a-b2bf-7684ab3ae629.png)

![image](https://user-images.githubusercontent.com/89985196/197544109-e9825434-2fd0-4e62-970b-3695b947ed8a.png)

Using C# Winforms here.

Having trouble resetting the chart completely. When resetting/clearing collections - I am left with relics from the previous chart (see image 1).

After adding the new series, the relics from an old plot still lingers.

Can't find documentation in the lvcharts.com repository. I'm guessing there's a method that I'm just not able to locate to .Clear() everything.

Thanks for developing this awesome tool by the way!

**Comment by En-JON-eer** [comment]
bump

**Reply by beto-rodriguez** [reply]
Hello, you should normally clear your data, and everything must erase automatically, how are you clearing your data? can I please know the steps to get this issue?

**Reply by En-JON-eer** [reply]
I allow the user to plot up to 10 different series on the same chart. I clear the ObservableCollection and nullify the global series objects. The problem I am seeing still is that the section lines are still visible when I load a different dataset. Ideally, cartesianChart1 should have a method to .ClearChartContents() or something similar. 

What do you think?
Jon
       
Code from my application:

        private delegate void ClearChartDelegate();
        private void ClearMainChart()
        {
            if (this.InvokeRequired)
            {
                this.Invoke(new ClearChartDelegate(ClearMainChart), new object[] { });
            }
            else
            {
                ChartSeriesCollection.Clear();
                ChartSeriesCollection = new ObservableCollection<ISeries>();
                this.cartesianChart1.Series = null;
                S1 = null;
                S2 = null;
                S3 = null;
                S4 = null;
                S5 = null;
                S6 = null;
                S7 = null;
                S8 = null;
                S9 = null;
                S10 = null;
            }
        }

The series objects are declared as such:

        private LineSeries<ObservableValue> S1 = null;
        private LineSeries<ObservableValue> S2 = null;
        private LineSeries<ObservableValue> S3 = null;
        private LineSeries<ObservableValue> S4 = null;
        private LineSeries<ObservableValue> S5 = null;
        private LineSeries<ObservableValue> S6 = null;
        private LineSeries<ObservableValue> S7 = null;
        private LineSeries<ObservableValue> S8 = null;
        private LineSeries<ObservableValue> S9 = null;
        private LineSeries<ObservableValue> S10 = null;
